uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestRightsTest.php

```diff
@@ -115,7 +115,7 @@ public function testCanChangeDatesForReturnsFalseForTOILAndLeaveRequestTypesWhen
       $managerRightsService->canChangeDatesFor($contactID, $status, LeaveRequest::REQUEST_TYPE_LEAVE)
     );
 
-    $this->assertFalse(
+    $this->assertTrue(
```

Comment:
I think this might not be quite accurate: `testCanChangeDatesForReturnsTrueForTOILAndLeaveRequestTypesWhenCurrentUserIsAdminIrrespectiveOfStatusPassed` since admin is now granted all edit access to all request types. So I broke down test as per user role and for admin the test is called now `testCanChangeDatesForReturnsTrueForAnyRequestTypeWhenCurrentUserIsAdminIrrespectiveOfStatusPassed`